optimade/models/entries.py

```diff
@@ -122,15 +129,17 @@ class EntryResource(Resource):
     - The entry of type `<type>` and ID `<id>` MUST be returned in response to a request for `/<type>/<id>` under the versioned base URL.
 
 - **Example**: `"structures"`""",
+        support=SupportLevel.MUST,
+        queryable=SupportLevel.MUST,
     )
 
-    attributes: EntryResourceAttributes = Field(
+    attributes: EntryResourceAttributes = StrictField(
```

Comment:
This is a MUST level field and OPTIONAL level query.
```suggestion
    attributes: EntryResourceAttributes = OptimadeField(
```

Reply:
I left attributes as a `StrictField` because it wasn't clear to me what the query level should be. Some fields inside attributes a MUST support queries, so surely attributes MUST too?

Reply:
Will also need to add descriptions to Structures->attributes and References->attributes in order to make them `OptimadeField`s

Reply:
Sure, but it _is_ a required field for entries. The query level can be set to the minimum, since it's not relevant.

Reply:
Example description: `"Dictionary of entity properties and OPTIMADE-specific fields"`.
There shouldn't be a need to change `StructuresResource` and `ReferencesResource` I think?

Reply:
> Sure, but it _is_ a required field for entries. The query level can be set to the minimum, since it's not relevant.

I might even go for `None` for query level...



> Example description: `"Dictionary of entity properties and OPTIMADE-specific fields"`.
> There shouldn't be a need to change `StructuresResource` and `ReferencesResource` I think?

I hoped not, as the definition is just a type hint so the rest might be inherited from `EntryResource`, but unfortunately it needs to be written out explicitly for it to appear in the model schemas. I just copied over the same generic description from `EntryResource`

Reply:
Reverted this for now, and will raise an issue as these suggested changes modify the schema in such a way that we currently can't handle